lib/network/ci-external-load-balancer.ts

```diff
@@ -64,6 +67,23 @@ export class JenkinsExternalLoadBalancer {
       },
     });
 
+    this.loadBalancer.logAccessLogs(props.accessLogBucket, accessLoggingPrefix);
```

Comment:
is there no option to store the logs in CW?

Reply:
No, only metrics of load balancer can be seen on cloudwatch https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-monitoring.html